.hound.yml

```diff
@@ -0,0 +1,2 @@
+rubocop:
```

Comment:
Happy to break this out to it's own PR but it fixes hound not using the custom rubocop configs (line length, etc)

Reply:
Totally fine with this, might add a few more exceptions and apply it to tove as well.